### DISCUSSION TITLE
Clarification about ExFat  and formatting from within Tesla UI

### DISCUSSION THREAD

**Discussion by altancode** [marcone/teslausb#814] [Q&A]
There's a note in the config file saying that ExFAT is not recommend.   However, the note also says that ExFAT is the file system used by Tesla when formatted in the car.

Can I safely conclude that when using TeslaUSB I should not format the TeslaUSB from the UI?

If that's correct, yet it somehow occurs, what's the best approach to recover?   Would it be:

1. ssh into the RPI (or whatever device)
2. sudo -i
3. rm /backingfiles/cam_disk.bin
4. /root/bin/setup-teslausb upgrade
5. Answer Yes when prompted about deleting snapshots

With full understanding this will erase all recordings?

Of course, the goal of the above is to get TeslaUSB to create the backing file and set it up for vfat (or whatever is recommended and apparently also supported by Tesla).

Thanks!

**Comment by marcone** [comment]
Yes, the steps you listed should recreate the disk image.
